### ISSUE TITLE
After Upgrade from 19.4.15. to any higher version the AdminMenu is missing

### ISSUE THREAD

**Issue by homerjsim** [OpenMage/magento-lts#3401]
### Preconditions (*)

1. Version 19.4.15

### Steps to reproduce (*)

1. Upload an overwrite all files from the archive
2. clear cache /var/cache and /var/session

### Expected result (*)

1. I can login to the backend
2. I see all menus

### Actual result (*)

1. I can login to the backend
2a) I only see the header and the global search  (Version 19.4.16)
2b) on a higher version I can see the dashboard but no adminmenu-bar

**Comment by pquerner** [comment]
Please attach more information about your server, such as PHP Version.
(Sounds like you have a php error and rendering is halted => you only get half a page)

**Comment by ADDISON74** [comment]
I installed magento-lts version 1.9.4.15 in a DDEV test environment (Apache, PHP 8.2, MariaDV). This is the screenshot of the Backend after installation

![screenshot-1](https://github.com/OpenMage/magento-lts/assets/8360474/9b34ad6e-9c30-47a7-a6bf-610b382869cb)

Then I copied all the files from magento-lts version 1.9.4.16 over 1.9.4.15. I did not flush the cache. This is the screenshot of the Backend

![screenshot-2](https://github.com/OpenMage/magento-lts/assets/8360474/709146d8-1a9b-4dd6-ae23-8fd493b72b5a)

As can be seen, the report is not confirmed. The causes that led to the situation shown can be many, if the installation has all the files, it is most likely from an extension that can be found by disabling it in app/etc/modules. By enabling logs you can get errors.

However, why do you upgrade to 1.9.4.16 when it is not the latest version, from more than a year ago?

**Comment by homerjsim** [comment]
OK, I will check all the extensions/modules if one of them breaks the amin menu on my site.
I will also activate error messages to see if this provides more clues what breaks my adminmenu. 

Regarding the question why I want to update to 19.4.16: I do not want to update to 19.4.16, I want to update to the newest 19.4.x. I tried that but there Have the same bug. So I restored everything from backup and started with version closest to mine to narrow it down to which changes in which version are incompatible with my installation.
